Sources/DeepLearning/Layer.swift

```diff
@@ -1282,8 +1282,8 @@ public extension RNNCell {
 
 /// A Simple RNN Cell.
 public struct SimpleRNNCell<Scalar: TensorFlowFloatingPoint>: RNNCell, VectorNumeric {
-    public var weight: Tensor<Scalar>
-    public var bias: Tensor<Scalar>
+    public var h_W: Tensor<Scalar>
```

Comment:
I think it would be easier to read if this was named “W_h” since this is how they are defined in math.